.github/workflows/maven.yml

```
@@ -0,0 +1,19 @@
+name: Tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

According to our experiences, a version change in future can break us, so better to fix it.